Description: Fix eigen name in pkgconfig deps
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Last-Update: 2019-01-28
Forwarded: No

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -64,7 +64,7 @@ set(DART_MINOR_VERSION "9")
 set(DART_PATCH_VERSION "2")
 set(DART_VERSION "${DART_MAJOR_VERSION}.${DART_MINOR_VERSION}.${DART_PATCH_VERSION}")
 set(DART_PKG_DESC "Dynamic Animation and Robotics Toolkit.")
-set(DART_PKG_EXTERNAL_DEPS "eigen, ccd, fcl, assimp, boost")
+set(DART_PKG_EXTERNAL_DEPS "eigen3, ccd, fcl, assimp")
 
 #===============================================================================
 # Build options
--- a/cmake/dart.pc.in
+++ b/cmake/dart.pc.in
@@ -9,4 +9,4 @@ Description: @DART_PKG_DESC@
 Version: @DART_VERSION@
 Requires: @DART_PKG_EXTERNAL_DEPS@
 Libs: -L${libdir} -ldart
-Cflags: @CMAKE_CXX_FLAGS@
+Cflags: -I${includedir}
