From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Sun, 14 Oct 2018 23:04:42 +0200
Subject: Fix find Flann

---
 cmake/FindFLANN.cmake | 27 +--------------------------
 1 file changed, 1 insertion(+), 26 deletions(-)

diff --git a/cmake/Findflann.cmake b/cmake/Findflann.cmake
index 804b904..675a852 100644
--- a/cmake/Findflann.cmake
+++ b/cmake/Findflann.cmake
@@ -20,24 +20,4 @@
 find_package(PkgConfig QUIET)
 
 # Check to see if pkgconfig is installed.
-pkg_check_modules(PC_FLANN flann QUIET)
-
-# Include directories
-find_path(FLANN_INCLUDE_DIRS
-    NAMES flann/flann.h
-    HINTS ${PC_FLANN_INCLUDEDIR}
-    PATHS "${CMAKE_INSTALL_PREFIX}/include")
-
-# Libraries
-find_library(FLANN_LIBRARIES flann_cpp
-    HINTS ${PC_FLANN_LIBDIR})
-
-# Version
-set(FLANN_VERSION ${PC_FLANN_VERSION})
-
-# Set (NAME)_FOUND if all the variables and the version are satisfied.
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(FLANN
-    FAIL_MESSAGE  DEFAULT_MSG
-    REQUIRED_VARS FLANN_INCLUDE_DIRS FLANN_LIBRARIES
-    VERSION_VAR   FLANN_VERSION)
+pkg_check_modules(FLANN flann QUIET)
