From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed, 18 Nov 2020 14:39:36 +0100
Subject: Fix SOVERSION to match upstream policy

Last-Update: 2019-02-14
Forwarded: https://github.com/dartsim/dart/pull/1239
---
 cmake/DARTMacros.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/DARTMacros.cmake b/cmake/DARTMacros.cmake
index e6f579f..35274e3 100644
--- a/cmake/DARTMacros.cmake
+++ b/cmake/DARTMacros.cmake
@@ -90,7 +90,7 @@ macro(dart_add_library _name)
   add_library(${_name} ${ARGN})
   set_target_properties(
     ${_name} PROPERTIES
-    SOVERSION "${DART_MAJOR_VERSION}.${DART_MINOR_VERSION}"
+    SOVERSION "${DART_MAJOR_VERSION}"
     VERSION "${DART_VERSION}"
   )
 endmacro()
