From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Fri Oct  8 16:06:49 CEST 2021
Subject: Fix examples compilation
Origin: https://github.com/dartsim/dart/pull/1545

diff --git a/examples/deprecated_examples/glut_human_joint_limits/CMakeLists.txt b/examples/deprecated_examples/glut_human_joint_limits/CMakeLists.txt
index f761d40..6a2b8af 100644
--- a/examples/deprecated_examples/glut_human_joint_limits/CMakeLists.txt
+++ b/examples/deprecated_examples/glut_human_joint_limits/CMakeLists.txt
@@ -38,7 +38,6 @@ if(DART_IN_SOURCE_BUILD)
       LINK_LIBRARIES ${required_libraries}
       COMPILE_FEATURES cxx_std_14
     )
-  endif()
   return()
 endif()
