From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Thu, 14 Sep 2023 11:55:37 +0000
Subject: skel test fail hard on armhf, mipsel and hangs in i386

Forwarded: no
Last-Update: 2022-01-25

Last-Update: 2022-01-25
---
 unittests/integration/test_SkelParser.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/unittests/integration/test_SkelParser.cpp b/unittests/integration/test_SkelParser.cpp
index adba5eb..5a36f9e 100644
--- a/unittests/integration/test_SkelParser.cpp
+++ b/unittests/integration/test_SkelParser.cpp
@@ -146,6 +146,7 @@ TEST(SkelParser, SerialChain)
   world->step();
 }
 
+#if !defined(__arm__) && !defined(__mips__) && !defined(__i386__)
 //==============================================================================
 TEST(SkelParser, VariousShapes)
 {
@@ -158,6 +159,7 @@ TEST(SkelParser, VariousShapes)
   for (auto i = 0u; i < 100; ++i)
     world->step();
 }
+#endif
 
 //==============================================================================
 TEST(SkelParser, RigidAndSoftBodies)
